### ISSUE TITLE
sfdx force:source:convert not working properly for child metadata (i.e. CustomField)

### ISSUE THREAD

**Issue by fechanique** [forcedotcom/cli#653]
### Summary

When you convert from a Package XML the behaviour is not the expected.
If you include for example the Account Object to convert from source to metadata the cli converts properly the whole object.
But if you include a Custom Field, it will only convert the first item for the object an nothing else.

### Steps To Reproduce:

Create a package xml with the following content:

```
<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
	<types>
		<members>Account</members>
		<name>CustomObject</name>
	</types>
	<types>
		<members>Account.MyField1__c</members>
		<members>Account.MiField2__c</members>
		<name>CustomField</name>
	</types>
	<version>50.0</version>
</Package>
```

run sfdx force:source:convert -x .\manifest\package.xml

### Expected result

You expect a Account.object xml field with the object metadata and 2 custom fields metadata.

### Actual result

You receive a Account.object xml field with only the metadata for the first field and noting else

### Additional information

**SFDX CLI Version**:
sfdx-cli/7.75.0-5521f40d11 win32-x64 node-v12.18.3

**SFDX plugin Version**:
@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.9.0 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/lwc-dev-server 2.5.1
├─ @oclif/plugin-help 2.2.3
└─ @oclif/plugin-update 1.3.10
@salesforce/sfdx-diff 0.0.6
@salesforce/sfdx-trust 3.4.3 (core)
alias 1.1.2 (core)
analytics 1.12.1 (core)
config 1.1.8 (core)
generator 1.1.3 (core)
salesforcedx 49.13.0 (core)
├─ templates 49.9.1 (core)
├─ salesforce-alm 49.12.1 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
├─ apex 0.1.0 (core)
└─ custom-metadata 1.0.10 (core)

**OS and version**:
win32-x64 node-v12.18.3

**Comment by cmduquer** [comment]
We are facing the same issue.

**Comment by johndturn** [comment]
I had the same problem, but after updating the CLI to v7.76.1, this was fixed for me. Looks like it was fixed over in #654.

**Comment by shetzel** [comment]
Yes, this was fixed with the release this past Saturday.  I'll search the github issues for duplicates and close them.  Thank you @johndturn
